### PR TITLE
Bug 1807131: Fix ripples for snackbar and alert dialogs

### PR DESCRIPTION
**Pull request by czlucius**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1807131

### CODE REVIEW

**Comment by czlucius** [comment]
Screenshot_20230314_154415_Firefox Fenix
Screenshot_20230314_152821_Firefox Fenix
Here are some screenshots, I forgot to put them earlier.
Thanks

**Comment by czlucius** [comment]
@csadilek would like to ask you on this. What do you think of this change?
Thanks

File: fenix/app/src/main/res/drawable/ripple_button.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```
Comment (Amejia481) [inline fenix/app/src/main/res/drawable/ripple_button.xml:4]: @czlucius I added the licenses to each files, also squashed the commits into a single one.

**Comment by czlucius** [comment]
Thanks

**Comment by czlucius** [comment]
@delia-pop I tested it on the update today (from the Play Store), and it seems to work on light mode as well. Can you confirm this behaviour? However, I'd like to file a new issue as the ripple seems to be off-centre, which did not happen when I was testing this change on my device before the patch was merged.
</img>
 
I can't post on Bugzilla since the issue is closed

**Comment by czlucius** [comment]
@delia-pop The above image is for `alert()`, bookmarks show the ripple just fine:
image

**Pull request merged**